examples/shared/message.go

```diff
@@ -0,0 +1,9 @@
+package shared
+
+import "github.com/yohamta/donburi"
+
+type TestMessage struct {
+	Message string
+}
+
+var TestMessageComponent = donburi.NewComponentType[TestMessage]()
```

Comment:
Not sure if this is actually required, I think just using the same `TestMessage` type is sufficient.

Reply:
Yeah just use the same actual type. There's a bit of code that generates a hash for the type name, so as long as they match it should work.